
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta name="Robots" content="index,follow" />
	<meta name="author" content="Luka Cvrk (www.solucija.com)" />
	<meta name="description" content="Description" />
	<meta name="keywords" content="key, words" />
	<link rel="stylesheet" type="text/css" href="css/screen.css" media="screen" />
	<title>Computational,Comparative,Evolutionary and Functional Genomics(CEFG)</title>
</head>
<body>
	<div id="content">
		<p id="top">Group of Computational&Comparative,Evolutionary and Functional Genomics(CEFG)</p>
		<div id="logo">
			<h1><a href="index.php">CEFG</a></h1>
		</div>
		<ul id="menu">
<li ><a href="index.php">Home</a></li>
			<li><a href="Introduction.html">Introduction</a></li>
			<li><a href="member.html">Member</a></li>
	 <li class="current"><a href="publication.html">Publication</a></li>
			<li><a href="service.html">Service</a></li>
			<li><a href="lecture.php">Lecture</a></li>
			<li><a href="news.html">News</a></li>
		</ul>
		<div class="line"></div>
	 <div class="introduction_third">
		<p></p><h3>Publications</h3> <p><h2>|<a href="#2015">2015</a> |<a href="#2014">2014</a> |<a href="#2013-2012">2013-2012</a> |<a href="#2011-2010">2011-2010</a> 
	|<a href="#2009-2007">2009-2007</a> |<a href="#2006-">2006-</a> |</h2></p>
					<div class="text">
					
				<a name = "2015"></a><p></p><h3>2015</h3>
				<h2>
               
                   
				   <p>48. Hua ZG#, Lin Y#, Yuan YZ, Yang DC, Wei W, Guo FB* (2015) ZCURVE 3.0: identify bacterial genes with higher accuracy as well as automatically and accurately select essential genes. <em>Nucleic Acids Res </em>. 43(W1):W85-90.
				   <a href="http://nar.oxfordjournals.org/content/early/2015/05/14/nar.gkv491.full?sid=924b93fe-01cd-4f55-a12a-5679c549b396">[Full Text]</a>.	</p>
				   
				   <p>47. Guo FB*, Ye YN, Ning LW, Wei W (2015) Three computational tools for predicting bacterial essential genes. <em>Methods in Molecular Biology</em>. 1279:205-217. <a href="http://link.springer.com/protocol/10.1007/978-1-4939-2398-4_13">[Full Text]</a>.	</p>
				   
				   <p>46. Zhao HL, Xia ZK, Hua ZG, Wei W* (2015) Review Selectional versus mutational mechanism underlying genomic features of bacterial strand asymmetry: a case study in Clostridium acetobutylicum. <em>Genet Mol Res</em>.  14(1):1911-1925. <a href = "http://www.geneticsmr.com/articles/4079">[Full Text]</a> </p>
                   
                    <p>45. Deng YY#, Guo FB* (2015) Applications of four machine learning algorithms in identifying bacterial essential genes based on composition features.<em>IEEE</em>.10.1109/ChinaSIP.2015.7230519 
				   <a href="http://ieeexplore.ieee.org/xpls/abs_all.jsp?arnumber=7230519&tag=1">[Full Text]</a>.	</p>
				   
				   <p><a href = "#TOP">[TOP]</a></p> 	
				 </h2>	
				
				 <a name="2014"></a><p></p><h3>2014</h3> 
				 	<h2>
				 		
				 		 <p>44. Wei W, Ning LW, Ye YN, Li SJ, Zhou HQ, Huang J, Guo FB* (2014) SMAL: A resource of spontaneous mutation accumulation lines. <em>Mol Biol Evol.</em> 31(5): 1302-1308. <a href="http://mbe.oxfordjournals.org/content/early/2014/02/14/molbev.msu073.short?rss=1">[Full Text]</a>. (IF>10).

</p>

				 		 <p>43. Zhou HQ?, Ning LW?, Zhang HX, Guo FB* (2014) Analysis of the relationship between genomic GC content and patterns of base usage, codon usage and amino acid usage in prokaryotes: similar GC content adopts similar compositional frequencies regardless of the phylogenetic lineages. <em>PLoS One.</em> 9(9):e107319. <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0107319">[Full Text]</a>.

</p>
		 <p>42. Wei W?, Ye YN?, Luo S, Deng YY, Lin D, Guo FB* (2014) IFIM: a database of integrated fitness information for microbial genes. <em>Database.</em> 2014: bau052. <a href="http://database.oxfordjournals.org/content/2014/bau052.full.pdf+html">[Full Text]</a>. 

</p>		 		
				 		

 <p>41. Guo FB, Lin Y, Chen LL* (2014) Recognition of protein-coding genes based on Z-curve algorithms. <em>Current Genomics.</em> 15(2): 95-103. <a href="http://benthamscience.com/journal/abstracts.php?journalID=cg&articleID=121236">[Full Text]</a></p> 
				 		<p>
			 	40. Ning LW, Lin H, Ding H, Huang J, Rao N, Guo FB*. (2014) 
			 	Predict bacterial essential genes using only sequence composition information. 
			 	<em>Genet Mol Res.</em> 13(2): 4564-4572.<a href="http://www.geneticsmr.com/year2014/vol13-2/pdf/gmr3515.pdf"> [Full Text]</a></p>

			 <p>39. Guo FB*, Xia ZK, Wei W, Zhao HL (2014) Statistical analyses of conserved features of genomic islands in bacteria. <em>Genet Mol Res.</em> 13(1): 1782-1793. <a href="http://www.geneticsmr.com/year2014/vol13-1/pdf/gmr2763.pdf">[Full Text]</a></p>		
				
				 <p>
			 38. Guo FB*. (2014) Editorial [Hot Topic]: Predict various types of genes in prokaryotic and eukaryotic genomes. <em>Curr Bioinform.</em> 9(2): 87-88.<a href="http://www.eurekaselect.com/120941/article"> [Full Text]</a></p>

			 
			 </p> 
			 		 </h2>
			 <p><a href="#top">[TOP]</a></p>	
		 <a name="2013-2012"></a><p></p><h3>2013-2012</h3> 

				<h2><p>37. Ye YN, Hua ZG, Huang J, Rao N, Guo FB*. (2013) CEG: a database of essential gene clusters.<em> BMC Genomics. </em>14: 769.<a href="http://www.biomedcentral.com/1471-2164/14/769"> [Full Text]</a>
<p>36. Wei W, Ning LW, Ye YN, Guo FB*. (2013) Geptop: a gene essentiality prediction tool for sequenced bacterial genomes based on orthology and phylogeny.<em> PLoS One. </em>8(8): e72343.<a href="http://dx.plos.org/10.1371/journal.pone.0072343"> [Full Text]</a>
			 </p>
			 					<p>35. Wei W, Zhang T, Lin D, Yang ZJ, Guo FB*. (2013) Transcriptional abundance is not the single force driving the evolution of bacterial proteins.<em> BMC Evol Biol.</em> 13: 162. <a href="http://www.biomedcentral.com/1471-2148/13/162">[Full Text]</a>
			 </p>
			 					<p>34. Lin D, Yin X, Wang X, Zhou P, Guo FB*. (2013) Re-annotation of protein-coding genes in the genome of <span style="font-style: italic">Saccharomyces cerevisiae</span> based on support vector machines.<em> PLoS One.</em> 8(7): e64477. <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0064477">[Full Text]</a>
			 </p>
			 					
			 		 	<p>
			 	33. Ding H, Guo SH, Deng EZ, Yuan LF, Guo FB, Huang J, Rao N, Chen W, Lin H*. (2013) 
			 	Prediction of Golgi-resident protein types by using feature selection technique. 
			 	<em>Chemometr Intell Lab.</em> 124: 9-13.

			 	<a href="http://www.sciencedirect.com/science/article/pii/S0169743913000385"> [Full Text]</a>
			 </p> 			
			 			
			 				 	<p>
			 	32. Lin H*, Ding C, Yuan LF, Chen W, Ding H, Li ZQ, Guo FB, Huang J, Rao NN. (2013) 
			 	Predicting subchloroplast locations of proteins based on the general form of Chou's pseudo amino acid composition: approached from optimal tripeptide composition. 
			 	<em>Int J Biomath.</em> 62(2): 1350003.
			
			 	<a href="http://www.worldscientific.com/doi/abs/10.1142/S1793524513500034"> [Full Text]</a>
			 </p> 
			 
			 					
			 					<p>31. Guo FB, Xiong L, Teng JL, Yuen KY, Lau SK*, Woo PC*. (2013) Re-annotation of protein-coding genes in 10 complete genomes of <span style="font-style: italic">Neisseriaceae</span> family by combining similarity-based and composition-based methods.<em> DNA Res. </em>20(3): 273-276. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3686433/">[Full Text]</a>
</p>
 					<p>30. Guo FB*, Ye YN, Zhao HL, Lin D, Wei W. (2012) Universal pattern and diverse strengths of successive synonymous codon bias in three domains of life, particularly among prokaryotic genomes.<em> DNA Res.</em> 19(6): 477-485. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23132389"></a><a href="http://dnaresearch.oxfordjournals.org/content/early/2012/11/06/dnares.dss027.long">[Full Text]</a>
			 </p>
 					<p>29. Guo FB*. (2012) Editorial [Hot Topic]: Replicating strand asymmetry in bacterial and eukaryotic genomes. <em>Curr Genomics.</em> 13(1): 2-3. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22942670"></a><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3269013/pdf/CG-13-2.pdf">[Full Text]</a>
			 </p>
 					<p>28. Guo FB*, Wei W. (2012) Prediction of genomic islands in three bacterial pathogens of pneumonia.<em> Int J Mol Sci.</em> 13(3): 416-421. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22002124"></a><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3317706/pdf/ijms-13-03134.pdf">[Full Text]</a>
			 </p>
 					<p>27. Guo FB*, Wei W, Wang XL, Lin H, Ding H, Huang J, Rao N. (2012) Co-evolution of genomic islands and their bacterial hosts revealed through phylogenetic analyses of 17 groups of homologous genomic islands. <em>Genet Mol Res.</em> 11(4): 3735-3743. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22002124"></a><a href="http://www.biomedexperts.com/Abstract.bme/23096693/Co-evolution_of_genomic_islands_and_their_bacterial_hosts_revealed_through_phylogenetic_analyses_of_17_groups_of_homol">[Full Text]</a>
		
			
				 	<p>
			 	26. Huang J*, Ru B, Zhu P, Nie F, Yang J, Wang X, Dai P, Lin H, Guo FB, Rao N. (2012) 
			 	MimoDB 2.0: a mimotope database and beyond. 
			 	<em>Nucleic Acids Res.</em> 40(Database issue): D271-D277.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/22053087">[PUBMED]</a>
			 	<a href="http://nar.oxfordjournals.org/content/40/D1/D271.long"> [Full Text]</a>
			 </p> 
			 
			 
			 
		
</h2>

<p><a href="#top">[TOP]</a></p>	
<a name="2011-2010"></a><p></p><h3>2011-2010</h3> 
<h2>

	 </p>
	 	<p>
			 	25. Guo FB*, Ning LW. (2011) 
			 	Strand-specific composition bias in bacterial genomes. 
			 	DNA Replication-Current Advances, Herve Seligmann (Ed.), ISBN: 978-953-307-593-8, InTech, DOI: 10.5772/18554.

			 	<a href="http://www.intechopen.com/books/howtoreference/dna-replication-current-advances/strand-specific-composition-bias-in-bacterial-genomes"> [Full Text]</a>
	    </p> 
					<p>24. Tong H, Guo FB*, Ye YN. (2011) Automatic prediction of non-coding RNA genes in prokaryotes based on compositional statistics. <em>Indian J Biochem Bio.</em> 48(6): 416-421. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22002124"></a><a href="http://nopr.niscair.res.in/handle/123456789/13251">[Full Text]</a>
			 </p>
			 <p>23. Ning L, Guo FB*. (2011) Theoretical analyses of gene expression for five human pathogens.<em> Afr J Microbiol Res.</em> 5(24): 4071-4079. <a href="http://www.academicjournals.org/ajmr/PDF/pdf2011/30Oct/Ning%20and%20%20Guo.pdf"> [Full Text]</a>
			 </p>

			 	<p>22. Wei W, Guo FB*. (2011) Prediction of genomic islands in seven human pathogens using the Z-Island method. <em>Genet Mol Res.</em> 10(4): 2307-2315. <a href="http://www.ncbi.nlm.nih.gov/pubmed/22002124">[PUBMED]</a><a href="http://www.geneticsmr.com/articles/1292"> [Full Text]</a>
			 </p>
			 <p>21. Du MZ, Guo FB*, Chen YY. (2011) Gene re-annotation in genome of the extremophile <span style="font-style: italic">Pyrobaculum aerophilum</span> by using bioinformatics methods.<em> J Biomol Struct Dyn.</em> 29(2): 391-401. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21875157">[PUBMED]</a><a href="http://www.jbsdonline.com/Gene-Re-annotation-in-Genome-of-the-Extremophile-iPyrobaculum-Aerophilum-i-by-Using-Bioinformatics-Methods-p18297.html1"> [Full Text]</a>
			 </p>
			 <p>
			 	20. Ru B, Huang J*, Dai P, Li S, Xia Z, Ding H, Lin H, Guo FB, Wang X. (2010) 
			 	MimoDB: a new repository for mimotope data derived from phage display technology.. 
			 	<em>Molecules.</em> 15(11): 8279-8288.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/21079566">[PUBMED]</a>
			 	<a href="http://www.mdpi.com/1420-3049/15/11/8279"> [Full Text]</a>
			 </p>
			 
			 	<p>
			 	19. Ding H, Liu L, Guo FB, Huang J, Lin H*. (2010) 
			 	Identify Golgi protein types with modified Mahalanobis discriminant algorithm and pseudo amino acid composition. 
			 	<em>Protein Pept Lett.</em> 18(1): 58-63.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/20955168">[PUBMED]</a>
			 	<a href="http://www.eurekaselect.com/87124/article"> [Full Text]</a>
			 </p>
			 	<P>18. Wei W, Guo FB*. (2010) Strong strand composition bias in the genome of <em>Ehrlichia canis</em> revealed by multiple methods.<em> Open Microbiol J.</em> 21: 98-102. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21253460">[PUBMED]</a><a href="http://related.springerprotocols.com/lp/pubmed-central/strong-strand-composition-bias-in-the-genome-of-ehrlichia-canis-y48UyyXPn0"> [Full Text]</a>
 </p>
			 	<p>17. Guo FB*, Ning LW, Huang J, Lin H, Zhang HX. (2010) Chromosome translocation and its consequence in the genome of <em>Burkholderia cenocepacia </em>AU-1054. <em>Biochem Biophys Res Commun.</em> 403(3-4): 375-379. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21078295">[PUBMED]</a><a href="http://linkinghub.elsevier.com/retrieve/pii/S0006-291X(10)02097-8"> [Full Text]</a>
			 </p>
			 	<p>
			 	16. Huang J*, Ru B, Li S, Lin H, Guo FB. (2010) 
			 	SAROTUP: scanner and reporter of target-unrelated peptides. 
			 	<em>J Biomed Biotechnol.</em> 2010: 101932.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/20339521">[PUBMED]</a>
			 	<a href="http://dx.doi.org/10.1155/2010/101932"> [Full Text]</a>
			 </p>

		<p>
			 	15. Lin H*, Ding H, Guo FB, Huang J. (2010) 
			 	Prediction of subcellular location of mycobacterial protein using feature selection techniques. 
			 	<em>Mol Divers.</em> 4(4): 667-671.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/19908156">[PUBMED]</a>
			 	<a href="http://dx.doi.org/10.1007/s11030-009-9205-1"> [Full Text]</a>
	    </p></h2>
<p><a href="#top">[TOP]</a></p>	
<a name="2009-2007"></a><p></p><h3>2009-2007</h3> 
<h2>
	<p>14. Guo FB*, Lin H, Huang J. (2009) A plot of G+C content against sequence length of 640 bacterial chromosomes shows the points are widely scattered in the upper triangular area. <em>Chromosome Res.</em> 17(3): 359-364. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19283496">[PUBMED]</a><a href="http://dx.doi.org/10.1007/s10577-009-9024-3"> [Full Text]</a>
	    </p>
		 <p>13. Guo FB* , Yuan JB. (2009) Codon usages of genes on chromosome, and surprisingly, genes in plasmid are primarily affected by strand-specific mutational biases in <em>Lawsonia intracellularis</em>. <em>DNA Res</em>. 16(2): 91-104. <a href="http://www.ncbi.nlm.nih.gov/pubmed/19221094">[PUBMED]</a><a href="http://dnaresearch.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=19221094"> [Full Text]</a>
	    </p>
			
	 <p>12. Guo FB* , Lin Y. (2009) Identify protein-coding genes in the genomes of <em>Aeropyrum pernix</em> K1 and <em>Chlorobium tepidum</em> TLS. <em>J Biomol Struct Dyn.</em> 26(4): 413-420.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/19108580">[PUBMED]</a><a href="http://www.jbsdonline.com/index.cfm?d=3035&c=4276&p=17019&do=detail"> [Full Text]</a>
	    </p>
	
			<p>
			 	11. Lin H*, Ding H, Guo FB, Zhang AY, Huang J. (2008) 
			 	Predicting subcellular localization of mycobacterial proteins by using Chou's pseudo amino acid composition. 
			 	<em>Protein Pept Lett.</em> 15(7): 739-744.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/18782071">[PUBMED]</a>
			 	<a href="http://www.eurekaselect.com/83072/article"> [Full Text]</a>
	    </p>
	
				 <p>10. Guo FB*, Yu XJ. (2007) Separate base usages of genes located on the leading and lagging strands in <em>Chlamydia muridarum</em> revealed by the Z curve method. <em>BMC Genomics.</em> 8: 366. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17925038">[PUBMED]</a><a href="http://www.biomedcentral.com/1471-2164/8/366"> [Full Text]</a></p>

			 <p>9. Guo FB*. (2007) The distribution patterns of bases of protein-coding genes, non-coding ORFs, and intergenic sequences in <em>Pseudomonas aeruginosa </em>PA01 genome and its implications. <em>J Biomol Struct Dyn.</em> 25(2): 127-133. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17718591">[PUBMED]</a><a href="http://www.jbsdonline.com/index.cfm?d=3028&c=4239&p=16115&do=detail"> [Full Text]</a></p>
				 <p>8. Guo FB*, Yu XJ. (2007) Re-prediction of protein-coding genes in the genome of <em>Amsacta moorei entomopoxvirus</em>. <em>J Virol Methods</em>. 146(1-2): 389-392. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17716751">[PUBMED]</a><a href="http://linkinghub.elsevier.com/retrieve/pii/S0166-0934(07)00265-0"> [Full Text]</a></p>

</h2>
<p><a href="#top">[TOP]</a></p>	
<a name="2006-"></a><p></p><h3>2006-</h3> 
			 
			<h2>
					<p>
			 	7. Guo FB, Zhang CT*. (2006) 
			 	ZCURVE_V: a new self-training system for recognizing protein-coding genes in viral and phage genomes. 
			 	<em>BMC Bioinformatics.</em> 10: 7-9.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/16401352">[PUBMED]</a>
			 	<a href="http://www.biomedcentral.com/1471-2105/7/9"> [Full Text]</a>
			 </p>
				<p>
			 	6. Guo FB, Wang J, Zhang CT*. (2004) 
			 	Gene recognition based on nucleotide distribution of ORFs in a hyper-thermophilic crenarchaeon, <em>Aeropyrum pernix K1</em>. 
			 	<em>DNA Res.</em> 11(6): 361-370.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/15871459">[PUBMED]</a>
			 	<a href="http://dnaresearch.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=15871459"> [Full Text]</a>
			 </p>
			 <p>
			 	5. Ou HY, Guo FB, Zhang CT*. (2004) 
			 	GS-Finder: a program to find bacterial gene start sites with a self-training method. 
			 	<em>Int J Biochem Cell Biol.</em> 36(3): 535-544.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/14687930">[PUBMED]</a>
			 	<a href="http://linkinghub.elsevier.com/retrieve/pii/S1357272503003078"> [Full Text]</a>
			 </p>
			 <p>
			 	4. Wang Y*, Ou H, Guo FB. (2003) 
			 	Recognition of translation initiation sites of eukaryotic genes based on an EM algorithm. 
			 	<em>J Comput Biol.</em> 10(5): 699-708.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/14633394">[PUBMED]</a>
			 	<a href="http://online.liebertpub.com/doi/abs/10.1089/106652703322539042"> [Full Text]</a>
			 </p>
			 <p>
			 	3. Ou HY, Guo FB, Zhang CT*. (2003) 
			 	Analysis of nucleotide distribution in the genome of Streptomyces coelicolor A3(2) using the Z curve method. 
			 	<em>FEBS Lett.</em> 540(1-3): 188-194.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/12681506">[PUBMED]</a>
			 	<a href="http://linkinghub.elsevier.com/retrieve/pii/S0014579303002631"> [Full Text]</a>
			 </p>
			 
				<p>
			 	2. Guo FB, Ou HY, Zhang CT*. (2003) 
			 	ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes. 
			 	<em>Nucleic Acids Res.</em> 31(6): 1780-1789.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/12626720">[PUBMED]</a>
			 	<a href="http://linkinghub.elsevier.com/retrieve/pii/S0006291X01961741"> [Full Text]</a>
			 </p>
			 
			 <p>
			 	1. Wang J*, Guo FB. (2002) 
			 	Base frequencies at the second codon position of Vibrio cholerae genes connect with protein function. 
			 	<em>Biochem Biophys Res Commun.</em> 290(1): 81-84.
			 	<a href="http://www.ncbi.nlm.nih.gov/pubmed/11779136">[PUBMED]</a>
			 	<a href="http://nar.oxfordjournals.org/cgi/pmidlookup?view=long&pmid=12626720"> [Full Text]</a>
			 </p> 

			</h2><p><a href="#top">[TOP]</a></p>	
			 <p>&nbsp;</p>
			 <p>&nbsp;</p>
			</div>
 </div>
		<div class="line"></div>
		<div id="footer">
			<p>&copy; Copyright CEFG(cefg.uestc.edu.cn) @ 2011 All rights reserved &minus; Modify: <a href="http://cefg.uestc.edu.cn" >CEFG</a>&minus;Design: Luka Cvrk, <a href="http://www.solucija.com" title="Free CSS Templates">Solucija</a>
				<p align="center">
				<!-- <script type="text/javascript" src="http://jh.revolvermaps.com/2/4.js?i=7kbiuol1g38&amp;m=0&amp;h=160&amp;c=ff0000&amp;r=0" async="async"></script>
			</p><p align="center"> Statistics start on March 15, 2014</p>
<noscript><a href="http://www.51.la/?16695035" target="_blank"><img alt="&#x6211;&#x8981;&#x5566;&#x514D;&#x8D39;&#x7EDF;&#x8BA1;" src="http://img.users.51.la/16695035.asp" style="border: none" /></a></noscript></p> //-->
		</div>	
	</div>
</body>
</html>